app/src/main/res/layout/fragment_feed.xml

```diff
@@ -1,56 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/frame_large">
+    android:background="@android:color/transparent">
```

Comment:
You can remove the "background" property if the fragment (or activity) has no background.